### PR TITLE
[ChainOps] Monitoring.

### PR DESCRIPTION
**Pull request by intl-man**
Integration of Halborn's logstash work.

### CODE REVIEW

File: deploy/helm/logstash/values.yaml [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+      }
+     }
+      filter {
+        mutate {
+          add_field => { "cluster" => '{{ .Values.logstash.args.cluster }}' }
+        }
+    }
+    output {
+      elasticsearch {
+        hosts => ["http://elk-prod-1125755348.us-east-1.elb.amazonaws.com:80"]
```
Comment (utx0) [inline deploy/helm/logstash/values.yaml:65]: do we want to setup a dns for this?

File: deploy/helm/logstash/values.yaml [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+    filter {
+      json {
+        source => "message"
+        remove_field => ["message"]
+        add_field => { "cluster" => '{{ .Values.logstash.args.cluster }}' }
+      }
+    }
+    output {
+      elasticsearch {
+        hosts => ["http://elk-prod-1125755348.us-east-1.elb.amazonaws.com:80"]
```
Comment (utx0) [inline deploy/helm/logstash/values.yaml:42]: should we setup a dns here too?

**Review by utx0 — APPROVED** [review]
Nice one bro.

File: deploy/helm/logstash/values.yaml (hunk 1 above)
Comment (intl-man) [reply deploy/helm/logstash/values.yaml:65]: Yep, once we've figured out the SSL issues we're seeing with logstash and elasticsearch. The kibana endpoint is actually available via a nice DNS entry already.

File: deploy/helm/logstash/values.yaml (hunk 2 above)
Comment (intl-man) [reply deploy/helm/logstash/values.yaml:42]: As above :)

**Review by utx0 — APPROVED** [review]

**Pull request merged**